core/lib/spree/core.rb

```diff
@@ -105,7 +105,6 @@ class GatewayError < RuntimeError; end
 
 require 'spree/i18n'
 require 'spree/localized_number'
-require 'spree/money'
```

Comment:
You might want to add a file with a deprecation warning, like we did in #6056 and friends.

Reply:
Done